jwt/api_jwt.py

```diff
@@ -100,6 +98,18 @@ def _validate_claims(self, payload, options, audience=None, issuer=None,
                           'Please use verify_exp in options instead.',
                           DeprecationWarning)
 
+        deprecated_requires = ['require_exp', 'require_iat', 'require_nbf']
+        verify_claims = set(deprecated_requires).intersection(options)
+        require_options = options.setdefault('require', [])
+        for opt in verify_claims:
+            opt_claim = opt[8:]
```

Comment:
Took me a couple of seconds to understand what this was. Can we do something like `opt.split('require_', 1)[1]`?